PyPortal_Word_of_the_Day/code.py

```diff
@@ -53,9 +56,18 @@
 pyportal.set_text("\nWord of the Day") # show title
 
 while True:
-    try:
-        value = pyportal.fetch()
-        print("Response is", value)
-    except RuntimeError as e:
-        print("Some error occured, retrying! -", e)
-    time.sleep(10*60) # Update every 10 minutes
+    if pyportal.touchscreen.touch_point:
+        try:
+            #set the JSON path here to be able to change between definition and example
+            pyportal._json_path=(WORD_LOCATION,
```

Comment:
I have no issues with accessing this path as it's not an exposed property and there's no other way of changing it.

* Before Line 62, add the following line: 
`# pylint: disable=protected-access`

Reply:
Got it!